Xlib/ext/xinput.py

```diff
@@ -170,12 +204,41 @@ def query_version(self):
         minor_version=0,
         )
 
+class Mask(rq.List):
+
+    def __init__(self, name):
+        rq.List.__init__(self, name, rq.Card32, pad=0)
+
+    def pack_value(self, val):
+
+        mask_seq = array.array(rq.struct_to_array_codes['L'])
+
+        if isinstance(val, integer_types):
+            # We need to build a "binary mask" that (as far as I can tell) is
+            # encoded in native byte order from end to end.  The simple case is
+            # with a single unsigned 32-bit value, for which we construct an
+            # array with just one item.  For values too big to fit inside 4
+            # bytes we build a longer array, being careful to maintain native
+            # byte order across the entire set of values.
+            if sys.byteorder == 'little':
+                fn = lambda v: mask_seq.insert(0, v)
```

Comment:
This is a QC minor warning: [Avoid assigning a lambda expression to a variable](https://www.quantifiedcode.com/app/issue_class/6803rnFX)